.github/workflows/ci.yml

```diff
@@ -52,10 +62,13 @@ jobs:
         sudo apt-get update
         sudo apt-get install libidn2-dev libcurl3-dev
         echo "STTP_NATIVE=1" >> $GITHUB_ENV
+    - name: Enable Loom-specific modules
+      if: matrix.java == '21'
+      run: echo "JDK_LOOM=1" >> $GITHUB_ENV
```

Comment:
or we can just use https://stackoverflow.com/questions/2591083/getting-java-version-at-runtime in the sbt build?

Reply:
This env variable makes the project build _only_ jdk21 servers. Checking java version in sbt instead of reading it might cause confusion in local development. For example, I use 21 as my default working JDK, but I still want to build the "JDK11 set".  More developers may have such a case. Maybe we should rename the variable to something meaning more "build loom servers only"?

Reply:
makes sense :) add this info to readme in `Contributing`?